src/layer/tile/GridLayer.js

```diff
@@ -101,7 +101,7 @@ L.GridLayer = L.Layer.extend({
 
 		// @option maxZoom: Number = undefined
 		// The maximum zoom level that tiles will be loaded at.
-//		maxZoom: undefined,
+		// maxZoom: undefined,
```

Comment:
We might want to remove the comment here, or else it will show up in the documentation as a continuation of the previous line.